cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/SsaInternals.qll

```diff
@@ -119,6 +214,12 @@ private newtype TDefOrUseImpl =
   TDirectUse(Operand operand, int indirectionIndex) {
     isUse(_, operand, _, _, indirectionIndex) and
     not isDef(true, _, operand, _, _, _)
+  } or
+  TIteratorUse(DirectUse use, DirectUse containerUse) {
+    IteratorDefUse::isIteratorUse(use, containerUse)
+  } or
+  TIteratorDef(DirectDef def, DirectUse containerUse) {
+    IteratorDefUse::isIteratorDef(def, containerUse)
```

Comment:
Nitpicking: this IPA type now goes: def-use-use-def, where I expect def-use-def-use